lib/generators/init-generator.js

```diff
@@ -246,28 +244,26 @@ module.exports = class InitGenerator extends Generator {
 						);
 						regExpForStyles = `${new RegExp(/\.css$/)}`;
 						if (this.isProd) {
-							ExtractUseProps = `
-								use: [{
-									loader: "style-loader"
-								},{
-									loader: "css-loader",
+							ExtractUseProps.push(
```

Comment:
Why this part is not a string anymore?

Reply:
With `extract-text-webpack-plugin` before, this part was used as an argument in `ExtractTextPlugin.extract({ ${ExtractUseProps} })` with `fallback`. Since it could not be represented as a js object, we used string to do it. The usage of `mini-css-extract-plugin` is changed, so we don't need to use string anymore.